api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -853,4 +854,97 @@ public Order discontinueOrder(Order orderToDiscontinue, String reasonNonCoded, D
 	 */
 	@Authorized(PrivilegeConstants.GET_ORDERS)
 	public List<OrderGroup> getOrderGroupsByEncounter(Encounter encounter) throws APIException;
+	
+	/**
+	 * Return all OrderGroupAttributeTypes
```

Comment:
Can you use proper english for the description instead of? `OrderGroupAttributeTypes`